modules/web-console/frontend/public/stylesheets/style.scss

```diff
@@ -1571,10 +1571,16 @@ th[st-sort] {
         .caret { margin-left: 3px; }
     }
 
-    a, i {
+    a {
```

Comment:
This might potentially break other markup. Are you sure these changes are really needed?

Reply:
It is style of chart configuration panel and does not used in other places.